### PR TITLE
Share button blog

### LINKED ISSUE #4022: Blog: Share Icons
share-th

With social media being the go-to place for resources, embedding a share button on blog posts will go a long way to increase web traffic. 

The share button should include the title of the post and should tag the official blazorise page on the particular social media platform.

### PR DESCRIPTION
**Pull request by ddjerqq**
## Added Share button blog

Closes #4022

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality not to work as expected)

## Checklist:

- [x] The PR is submitted to the correct branch (`master` for dev, `rel-1.x` for maintenance).
- [x] My code follows the code style of this project.
- [ ] I've added relevant tests.

### CODE REVIEW

File: Documentation/Blazorise.Docs.Server/wwwroot/img/blog/2024-05-17/how-to-create-social-media-share-buttons.png
Comment (stsrki) [inline Documentation/Blazorise.Docs.Server/wwwroot/img/blog/2024-05-17/how-to-create-social-media-share-buttons.png:1]: We will provide you with a blog banner image.

File: Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
...
+Are you ready to sprinkle some Blazorise magic into your Blazor app?<br/>
+Adding share buttons for social media platforms can give your users an easy<br/>
+way to spread the word about your awesome content. It's easier than you think, thanks to Blazorise!<br/>
+
+---
+
+## Let's dive in and jazz up your app with these fantastic buttons.
+
+## Install Blazorise
+```bash
```
Comment (stsrki) [inline Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md:25]: For proper formating please add an empty line between blocks and headings.

**Review by stsrki — COMMENTED** [review]
I forgot to mention. Blogs go into current production branch(rel-1.5) so that we can release them without breaking documentation version. I will rebase it now.

**Comment by stsrki** [comment]
It seems rebasing is not an option now. I assume you have created your feature branch from `master`? It is best to close the PR and recreate it starting from `rel-1.5` branch, not the `master`.

File: Documentation/Blazorise.Docs.Server/wwwroot/img/blog/2024-05-17/how-to-create-social-media-share-buttons.png
Comment (ddjerqq) [reply Documentation/Blazorise.Docs.Server/wwwroot/img/blog/2024-05-17/how-to-create-social-media-share-buttons.png:1]: Ah! okay thank you! I didn't know what to do so I improvised and made something, but I will keep this in mind for future

File: Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md (hunk 1 above)
Comment (ddjerqq) [reply Documentation/Blazorise.Docs/Pages/Blog/2024-05-17_HowToCreateSocialMediaShareButtons/Index.md:25]: I will do that

**Comment by ddjerqq** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by ddjerqq** [comment]
> I forgot to mention. Blogs go into current production branch(rel-1.5) so that we can release them without breaking documentation version. I will rebase it now.

Will consider this for future blogs! Thanks for the heads up!

**Comment by stsrki** [comment]
> > I forgot to mention. Blogs go into current production branch(rel-1.5) so that we can release them without breaking documentation version. I will rebase it now.
> 
> Will consider this for future blogs! Thanks for the heads up!

It needs to also go now into rel-1.5 since master is only for new development.

I will close the PR and you can open new that targets the right branch.

**Pull request closed without merging**